### PR TITLE
Add Homebridge

### PR DESCRIPTION
**Pull request by dragonandy**
# Pull request

**Purpose**
Add Homebridge as a new application

**Approach**
Add Homebridge 

**Open Questions and Pre-Merge TODOs**
Check all boxes as they are completed

- [x] Use github checklists. When solved, check the box and explain the answer.

**Learning**
add Homebridge  application
add Homebridge  docs
tested on my own machine and works fine

**Requirements**
Check all boxes as they are completed

- [x] These changes meet the standards for [contributing](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: compose/.apps/homebridge/homebridge.hostname.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+services:
+  homeassistant:
```
Comment (nemchik) [inline compose/.apps/homebridge/homebridge.hostname.yml:2]: Wrong name :)

File: compose/.apps/homebridge/homebridge.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+services:
+  homebridge:
+    container_name: homebridge
+    environment:
+      - TZ=${TZ}
+      - PGID=1000
+      - PUID=1000
+      - HOMEBRIDGE_CONFIG_UI=1
+      - HOMEBRIDGE_CONFIG_UI_PORT=${HOMEBRIDGE_PORT_8581}
+      - ENABLE_AVAHI=0
```
Comment (nemchik) [inline compose/.apps/homebridge/homebridge.yml:10]: Please sort these alphabetically. Also may as well make them all use variables.

File: compose/.apps/homebridge/homebridge.aarch64.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+services:
+  homebridge:
+    image: oznu/homebridge:ubuntu
```
Comment (nemchik) [inline compose/.apps/homebridge/homebridge.aarch64.yml:3]: Any particular reason for using the Ubuntu tag instead of the latest tag?

**Review by nemchik — CHANGES_REQUESTED** [review]

File: compose/.apps/homebridge/homebridge.aarch64.yml (hunk 3 above)
Comment (dragonandy) [reply compose/.apps/homebridge/homebridge.aarch64.yml:3]: This application use ubuntu based image to work better with some plugins. some plugin dont like alpine enviroment, furthermore alpine is not notably smaller than the ubuntu one.

Comment (dragonandy) [reply compose/.apps/homebridge/homebridge.aarch64.yml:3]: This application use ubuntu based image to work better with some plugins. The latest tag one is using alpine based image, but some plugin don't like the alpine enviroment, furthermore alpine is not notably smaller than the ubuntu one.

File: compose/.apps/homebridge/homebridge.hostname.yml (hunk 1 above)
Comment (dragonandy) [reply compose/.apps/homebridge/homebridge.hostname.yml:2]: thats a problem :(

**Comment by dragonandy** [comment]
I cleanup those problems, and force pushed two new commits. please do another round of review

**Review by nemchik — APPROVED** [review]
LGTM, thanks!

**Comment by nemchik** [comment]
Actually, I'm just now looking at the checks and it seems like there's some trailing whitespace issues, and we need a link added to the mkdocs.yml in the top of the repo.

**Comment by angrycuban13** [comment]
@dragonandy 

1. You need to fix whitespace in the following lines.
- 	9: Trailing whitespace
- 	10: Trailing whitespace
- 	11: Trailing whitespace

2. You need to add `homebridge.md` in `mkdocs.yml`

**Pull request closed without merging**